### PR TITLE
real fix, I hope

### PR DESCRIPTION
**Pull request by SimonR99**
we don't need pre build stage now since the gazebo bridge is now pre compiled

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -22,17 +26,41 @@ RUN apt update && apt upgrade -y
...
+RUN apt-get update && apt-get install -y \
+    software-properties-common \
+    wget \
+    lsb-release \
+    gnupg \
+    curl
+
+# Install Gazebo
+RUN wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg
+RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null
```
Comment (Leuchak) [inline .devcontainer/Dockerfile:39]: no need to use `sudo tee`, use only `tee`

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -22,17 +26,41 @@ RUN apt update && apt upgrade -y
...
-RUN echo "source /opt/ros/${ROS_DISTRO}/setup.bash" >> ~/.bashrc && \
-    echo "source /gazebo/gazebo_ws/install/setup.bash" >> ~/.bashrc
+RUN echo "source /opt/ros/${ROS_DISTRO}/setup.bash" >> ~/.bashrc
+
+COPY --chown=$USERNAME ./ /workspace/$USERNAME
 
 # Install ROS dependencies
 RUN rosdep update
 # Copy to preload the ros packages
-COPY ./ /workspace/$USERNAME/
```
Comment (Leuchak) [inline .devcontainer/Dockerfile:65]: the change has already been made by @IliTheButterfly

**Review by Leuchak — CHANGES_REQUESTED** [review]

**Review by SimonR99 — COMMENTED** [review]
fix sudo

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (SimonR99) [reply .devcontainer/Dockerfile]: fine

**Review by Leuchak — APPROVED** [review]

**Pull request merged**